src/presentation_exchange.rs

```diff
@@ -84,6 +84,16 @@ pub enum ConstraintsLimitDisclosure {
     Preferred,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub enum VerifiableFormat {
+    #[serde(rename = "jwt_vc_json")]
+    JwtVcJson,
+    #[serde(rename = "jwt_vp_json")]
+    JwtVpJson,
+    #[serde(rename = "ldp_vc")]
+    LdpVc,
+}
+
```

Comment:
Are there any other formats we'd like to include here?

Should `jwt_vp` and `jwt_vc` be separate options in addition to these?

Reply:
NOTE: I can also stash these changes and apply to another PR if preferable.

Reply:
This is fine, but please add `Other(String)` to capture other variants.

Reply:
Could you also reuse this enum as the key for the maps [here](https://github.com/spruceid/oid4vp-rs/blob/main/src/core/metadata/parameters/wallet.rs#L137)
and [here](https://github.com/spruceid/oid4vp-rs/blob/main/src/core/metadata/parameters/verifier.rs#L8).

Reply:
Commit [1a2af67](https://github.com/spruceid/oid4vp-rs/pull/8/commits/1a2af6719c47bd8827f43504c0048480e8d273db) adds a designation for `jwt_vc_json` and `jwt_vp_json` with the wallet supported types claim format.

Since technically, its a presentation format more so than a claim format, it might make sense to break this into a separate type, or rename `ClaimFormat` to something more generic, possibly `Format` and `FormatDesignation`.

I'm open to suggestions on the naming and whether it makes sense for it to belong where I put it in the above commit.